### ISSUE TITLE
Confirm noise rank on - no emptyroom datasets

### ISSUE THREAD

**Issue by jstout211** [jstout211/enigma_MEG#33]
In commit fbdf2e751b7e2471fccc237717ada2c2a7d9457f, the below change was made.  @nugenta - confirm this is correct.

-                    inversion='matrix', weight_norm='unit-noise-gain', rank=noise_rank)
+                    inversion='matrix', weight_norm='unit-noise-gain', rank=epo_rank)

**Comment by nugenta** [comment]
Yes, this is correct - it is changing the rank in the condition where there is no emptyroom and thus no noise_rank to be calculated.
